torch/csrc/jit/passes/onnx/pattern_conversion/pattern_conversion.cpp

```diff
@@ -146,7 +146,7 @@ std::unordered_map<int64_t, ConvertedIndex> MergeSliceAndSelectToIndices(
           std::forward_as_tuple(index_tensor, aten::select));
       dim_offset++;
     } else {
-      AT_ERROR(
+      TORCH_INTERNAL_ASSERT(
```

Comment:
```suggestion
      TORCH_CHECK(false,
```

Use `TORCH_CHECK` for user input error. Also note the first argument of both `TORCH_CHECK` and `TORCH_INTERNAL_ASSERT` is for condition. `AT_ERROR` on the other hand only takes message string as argument.

Reply:
Done